ERCS/erc-7540.md

```diff
@@ -224,30 +310,107 @@ MUST be emitted when a redemption Request is submitted using the `requestRedeem`
   type: event
 
   inputs:
-    - name: sender
-      indexed: true
-      type: address
-    - name: operator
+    - name: receiver
       indexed: true
       type: address
     - name: owner
       indexed: true
       type: address
+    - name: rid
+      indexed: true
+      type: uint256
+    - name: sender
```

Comment:
Why is `sender` here towards the end, compared to the `DepositRequest` event?

Reply:
I fixed depositRequest to use the same order